specification/definitions/CarrierContract.json

```diff
@@ -14,32 +14,24 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
-        "groups": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ServiceGroup"
-          }
-        },
-        "options": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ServiceOption"
-          }
-        },
-        "insurances": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/ServiceInsurance"
-          }
+        "currency": {
```

Comment:
Question: with this new setup, do we still need the currency in the regions table?

Reply:
We never needed a currency in the regions table, did we? I thought it was there so users would know what the default currency in a region is.